config/amip_configs/amip_edonly.yml

```diff
@@ -0,0 +1,29 @@
+FLOAT_TYPE: "Float32"
+albedo_model: "CouplerAlbedo"
+atmos_config_file: "config/longrun_configs/amip_target_edonly.yml"
+checkpoint_dt: "720hours"
+co2: "maunaloa"
+coupler_toml: ["toml/amip.toml"]
```

Comment:
This file has many parameters you don't need, but it shouldn't be a problem I think.